internal/receiver/scriptedinputsreceiver/config.go

```diff
@@ -38,7 +38,7 @@ const (
 
 var availableScripts = func() []string {
 	var s []string
-	for _, sn := range scripts {
+	for sn, _ := range scripts {
```

Comment:
lint failures:

```suggestion
	for sn := range scripts {
```

apologies for this error, `scripts` used to be a slice when I first spiked the organization update.